index.js

```diff
@@ -1,17 +1,49 @@
 /* eslint-env node */
 'use strict';
 
-const react = require('broccoli-react');
+const configureJsxTransform = require('./lib/configure-jsx-transform');
+const mergeTrees = require('broccoli-merge-trees');
+const Funnel = require('broccoli-funnel');
+const path = require('path');
 
 module.exports = {
   name: 'ember-cli-react',
 
-  preprocessTree: function(type, tree) {
```

Comment:
Actually we should be able to continue using `preprocessTree` here. Instead of using broccoli-react we do babel transpile and feed into funnel here directly.